libsolidity/codegen/ir/IRGenerationContext.h

```diff
@@ -112,6 +115,8 @@ class IRGenerationContext
 	/// function call that was invoked as part of the try statement.
 	std::string trySuccessConditionVariable(Expression const& _expression) const;
 
+	std::set<ContractDefinition const*>& subObjects() { return m_subObjects; }
```

Comment:
Should we call these `Objects`?

Reply:
We can call them objects, I wanted to be more specific with this, but no point actually

Reply:
I mean we could use the more specific term - objects is just the thing used in Yul terms where something like a contract does not exist.